stable/mission-control/README.md

```diff
@@ -330,6 +389,11 @@ The following table lists the configurable parameters of the mission-control cha
 | `postgresql.slave.nodeSelector`              | PostgreSQL slave node selector                  | `{}`                                  |
 | `postgresql.slave.affinity`                  | PostgreSQL slave node affinity                  | `{}`                                  |
 | `postgresql.slave.tolerations`               | PostgreSQL slave node tolerations               | `[]`                                  |
+| `common.customInitContainers`                | Custom init containers                          | ` `                                   |
```

Comment:
Add `see values.yaml` as the default please for all of these

Reply:
Done